### PR TITLE
dli queue support scale

### LINKED ISSUE #1347: [DLI] Add scaling support; add import support
### Use-cases

[DLI] Add scaling support; add import support

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
Dli queue service  support scale and import 

**Which issue this PR fixes**:
fixes #1347

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dli' TESTARGS='-run=TestAccDliQueue_basic'

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dli -v -run=TestAccDliQueue_basic -timeout 360m -parallel 4
=== RUN   TestAccDliQueue_basic
--- PASS: TestAccDliQueue_basic (566.08s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dli       566.136s
```

### CODE REVIEW

File: docs/resources/dli_queue.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ The following arguments are supported:
 * `description` - (Optional, String, ForceNew) Description of a queue. 
     Changing this parameter will create a new resource.
 
-* `cu_count` - (Required, Int, ForceNew) Minimum number of CUs that are bound to a queue. The value can be 16,
-  64, or 256. Changing this parameter will create a new resource.
+* `cu_count` - (Required, Int ) Minimum number of CUs that are bound to a queue. Initial value can be `16`,
```
Comment (ShiChangkuo) [inline docs/resources/dli_queue.md]: format error: extra spaces

File: docs/resources/dli_queue.md [hunk 2]
```diff
@@ -76,3 +76,10 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - Specifies a resource ID in UUID format.
 
 * `create_time` -  Time when a queue is created.
+
+## Import
+
+queue can be imported by  `id`. For example,
```
Comment (ShiChangkuo) [inline docs/resources/dli_queue.md]: s/queue/DLI queeus/

File: huaweicloud/services/dli/resource_huaweicloud_dli_queue.go [hunk 3]
```diff
@@ -249,3 +272,57 @@ func resourceDliQueueDelete(d *schema.ResourceData, meta interface{}) error {
...
+	client, err := config.DliV1Client(config.GetRegion(d))
+	if err != nil {
+		return fmtp.Errorf("error creating DliV1Client: %s", err)
+	}
+	opt := queues.ActionOpts{
+		QueueName: d.Id(),
+	}
+	if d.HasChange("cu_count") {
+		oldValue, newValue := d.GetChange("cu_count")
+		opt.CuCount = newValue.(int) - oldValue.(int)
```
Comment (ShiChangkuo) [inline huaweicloud/services/dli/resource_huaweicloud_dli_queue.go]: Can this value be negative？

Comment (hmbig2) [reply huaweicloud/services/dli/resource_huaweicloud_dli_queue.go]: add more validate rule

**Comment by hmbig2** [comment]
please add scale in test

**Pull request merged**